server/modules/surveyRdb/schemaRdb/dataView.js

```diff
@@ -23,91 +23,145 @@ export const columns = {
   keys: '_keys',
 }
 
-export const getColUuid = nodeDef => `${NodeDef.getName(nodeDef)}_${DataTable.colNameUuuid}`
+export const getColUuid = (nodeDef) => `${NodeDef.getName(nodeDef)}_${DataTable.colNameUuuid}`
+
+/**
+ * Returns the list of column names relative to the specified node def
+ * (columns relative to its children and to its ancestors' children).
+ *
+ * @param {!object} survey - The survey.
+ * @param {!object} nodeDef - The context node definition.
+ * @returns {Array} List of column names.
+ */
+export const getNodeDefColumnNames = (survey, nodeDef) => {
+  const getNodeDefCols = (nodeDefCurrent, isSelf) => {
```

Comment:
isSelf can be calculated directly in this function, it doesn't need to be passed

Reply:
right